chronicle-front/src/app/components/login/login.component.spec.ts

```diff
@@ -31,13 +31,13 @@ describe('LoginComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
-  });
-
-  it('should render firebase-ui login form', () => {
-    const firebaseUi = fixture.debugElement.nativeElement.querySelector('firebase-ui');
-    expect(firebaseUi).toBeDefined();
-  });
+  // it('should create', () => {
```

Comment:
Again, please disable the tests properly if you are having problems with them.

Reply:
Uncommented the tests, disabled them by using 'xif'.